.prettierignore

```diff
@@ -3,4 +3,4 @@ dist
 node_modules
 *.log
 yarn.lock
-flow-typed/*.generated.*
\ No newline at end of file
+flow-typed/*.generated.*
```

Comment:
Not sure why this changed?

Reply:
It's a newline, my editor auto adds by default

Reply:
same as above

Reply:
Removed